src/site/filters/liquid.js

```diff
@@ -750,4 +750,46 @@ module.exports = function registerFilters() {
     // Encode the string.
     return he.encode(stringWithoutSingleQuotes, { useNamedReferences: true });
   };
+
+  // fieldCcVetCenterFeaturedCon data structure is different
+  // from objects inside fieldVetCenterFeatureContent. Recreates the array
+  // with the expected structure so that it can be directly passed inside the template
+  liquid.filters.appendCentralizedFeaturedContent = (
+    ccFeatureContent,
+    featureContentArray,
+  ) => {
+    if (!ccFeatureContent || !ccFeatureContent.fetched)
+      return featureContentArray;
+    /* eslint-disable camelcase */
+    const {
+      field_description,
+      field_section_header,
+      field_cta,
+    } = ccFeatureContent.fetched;
+    const featureContentObj = {
+      entity: {
+        fieldDescription: {
+          processed: field_description[0].processed,
```

Comment:
I suggest using optional chaining here in case `field_description` is empty:

```suggestion
          processed: field_description[0]?.processed,
```